athletics.lic

```diff
@@ -231,10 +231,14 @@ class Athletics
 
   def shard_athletics
     until done_training?
-      if UserVars.athletics > 20 && UserVars.athletics <= 90
-        swim_zaldeni
-      elsif UserVars.athletics < 240
-        override_location_and_practice('shard_embrasure')
+      if UserVars.athletics < 240
```

Comment:
Should this be an `elsif`? I'm not sure of the code implications but this starts a new if/elsif block at this point.

Reply:
No, the stuff in red is what is being removed. :)

Reply:
Ah hell LOL. My fault. You can tell I'm tired.